### ISSUE TITLE
still some docker-storage-setup* files present

### ISSUE THREAD

**Issue by lsm5** [projectatomic/container-storage-setup#211]
I see there are still 3 docker-storage-setup* files present in the repo. Any reason for not doing the rename for those? Too docker specific?

Also, should the new container-storage-setup rpm need a c-s-s.service unitfile? Or is the d-s-s.service file only relevant for docker?

**Comment by rhatdan** [comment]
No the idea is the docker* files will be shipped as part of the docker.spec file, they are really just examples in container-storage-setup.  OCID will have similar files that it ships.  container-storage-setup is not a service it is just a script that other services can use so there should NOT be a c-s-s.service.

**Comment by lsm5** [comment]
can we move the docker* files to `@projectatomic/docker/contrib` in that case? Just that having to fetch the same source tarball in both docker and container-storage-setup sounds like trouble.

**Comment by rhatdan** [comment]
@rhvgoyal WDYT?

@lsm5 We wanted to move slowly so that people using docker-storage-setup from source (If there are any) could continue to use container-storage-setup. The goal is a make install of this would work with a previous version of docker.service.

**Comment by lsm5** [comment]
hmm, in that case we could keep copies of those files in both docker and in here, and then get rid of them from in here when we're reasonably certain it's safe.

**Comment by rhatdan** [comment]
Sure in the long run they should just be in the docker package repo.

**Comment by rhvgoyal** [comment]
Yes, let us move little slow for now. I want to make sure "make install" works reasonably well for docker users.
